docs/Guides/Ecosystem.md

```diff
@@ -635,7 +637,9 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+
```

Comment:
```suggestion
```

Reply:
I thought that `Community Tools` should be the same as `Community` and `Core` headings (that have empty lines around) but i will drop them as suggested.